File: treecEva_JSON/SL/scripts/temp_code/SL-MIX-S383.cpp
#define _USE_MATH_DEFINES
#include <iostream>
#include <memory>
#include <vector>
#include <numeric>

class PrimeNode {
public:
    int value;
    std::shared_ptr<PrimeNode> next;
    PrimeNode(int val) : value(val), next(nullptr) {}
};

int gcd(int a, int b) {
    while (b != 0) {
        int temp = b;
        b = a % b;
        a = temp;
    }
    return a;
}

int main() {
    // Create linked list of first 5 primes: 2, 3, 5, 7, 11
    auto head = std::make_shared<PrimeNode>(2);
    head->next = std::make_shared<PrimeNode>(3);
    head->next->next = std::make_shared<PrimeNode>(5);
    head->next->next->next = std::make_shared<PrimeNode>(7);
    head->next->next->next->next = std::make_shared<PrimeNode>(11);
    
    int secure_token_value = 0;
    std::vector<int> processed_primes;
    
    auto current = head;
    while (current != nullptr) {
        int prime = current->value;
        processed_primes.push_back(prime);
        
        // For each new prime, compute GCD with all previous primes
        for (size_t i = 0; i < processed_primes.size() - 1; ++i) {
            secure_token_value += gcd(prime, processed_primes[i]);
        }
        
        current = current->next;
    }
    
    std::cout << "Result: " << secure_token_value << std::endl;
    return 0;
}

File: treecEva_JSON/SL/scripts/temp_code/SL-MIX-S383.py
def is_prime(n):
    if n <= 1:
        return False
    for i in range(2, int(n**0.5) + 1):
        if n % i == 0:
            return False
    return True

primes = []
for num in range(2, 30):
    if is_prime(num):
        primes.append(num)

transform = lambda x: x * 2
transformed_primes = list(map(transform, primes))

from math import gcd
final_sum = 0
for p in transformed_primes:
    final_sum += gcd(p, 15)

print(f"Result: {final_sum}")